teamsyncd/teamsync.cpp

```diff
@@ -114,24 +114,26 @@ void TeamSync::onMsg(int nlmsg_type, struct nl_object *obj)
         return;
     }
 
+    unsigned int mtu = rtnl_link_get_mtu(link);
     addLag(lagName, rtnl_link_get_ifindex(link),
            rtnl_link_get_flags(link) & IFF_UP,
-           rtnl_link_get_flags(link) & IFF_LOWER_UP);
+           rtnl_link_get_flags(link) & IFF_LOWER_UP, mtu);
 }
 
 void TeamSync::addLag(const string &lagName, int ifindex, bool admin_state,
-                      bool oper_state)
+                      bool oper_state, unsigned int mtu)
 {
     /* Set the LAG */
     std::vector<FieldValueTuple> fvVector;
     FieldValueTuple a("admin_status", admin_state ? "up" : "down");
     FieldValueTuple o("oper_status", oper_state ? "up" : "down");
+    FieldValueTuple m("mtu", to_string(mtu));
     fvVector.push_back(a);
     fvVector.push_back(o);
```

Comment:
push_back(m)? :-)

Reply:
Oh.. missed it.. good find and thanks for fixing Ying.